test/cpp/qps/usage_timer.cc

```diff
@@ -69,6 +69,9 @@ static void get_cpu_usage(unsigned long long* total_cpu_time,
     }
   }
 #else
+  // Use the parameters to avoid unused-parameter warning
+  (void)total_cpu_time;
+  (void)idle_cpu_time;
```

Comment:
TIL.

Reply:
Note that my LinkedIn profile refers to me as "developer/educator" . Happy to be both at the same time.